nico.nim

```diff
@@ -736,6 +736,9 @@ proc ditherADitherXor*(v: float32, a = 149,b = 1234, c = 511) =
   gDitherADitherC = c
 
 proc ditherPass(x,y: int): bool {.inline.} =
+  if gDitherMode == DitherNone:
+    return true
+    
```

Comment:
```
of DitherNone:
    return true
```
is now dead code

Reply:
Yes, I know, but just removing it does not work since `case` has to cover every value. To make the compiler happy one would have to add `else: discard`, but then it would be possible to miss a value, if it got extended in the future. Not sure, what is more pretty.

Reply:
I think it's fine with the dead code, maybe a comment to say it's dead code would be best.